src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  margin: 0;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
Thank You for your response!
I`ll fix everything ))